### PR TITLE
feat(university-gateway): English titles for organizations, Using english titles in frontend of háskólanám

### PR DESCRIPTION
**Pull request by Ballioli**
Adding to the graphql the ability to fetch the english titles of organizations, updating frontend of haskolanam to use english tiles of universities

# ...

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs
image
image

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Review by jonarnarbriem — APPROVED** [review]

**Review by Toti91 — APPROVED** [review]
Codeowned LGTM

File: apps/web/screens/UniversitySearch/LandingPage.tsx [hunk 1]
```diff
@@ -59,13 +59,15 @@ const LandingPage: Screen<LandingPageProps> = ({
...
     const newArray = [...universities]
     newArray.sort((x, y) => {
-      const titleX = x.contentfulTitle || ''
-      const titleY = y.contentfulTitle || ''
-      return titleX.localeCompare(titleY)
+      const titleX =
+        locale === 'is' ? x.contentfulTitle || '' : x.contentfulTitleEn || ''
+      const titleY =
+        locale === 'is' ? y.contentfulTitle || '' : y.contentfulTitleEn || ''
+      return titleX.localeCompare(titleY, locale)
```
Comment (RunarVestmann) [inline apps/web/screens/UniversitySearch/LandingPage.tsx:66]: Perhaps using the sortAlpha function instead would help out with having "á" after "a" and so forth (not sure though if that even is a problem but just wanted to mention it

File: libs/cms/src/lib/cms.contentful.service.ts [hunk 2]
```diff
@@ -216,6 +216,7 @@ export class CmsContentfulService {
 
   async getOrganizationTitles(
     organizationKeys: string[],
+    locale?: 'en' | 'is',
```
Comment (RunarVestmann) [inline libs/cms/src/lib/cms.contentful.service.ts:219]: Isn't there a better type that could be used here instead of 'en' | 'is' ?

**Review by RunarVestmann — APPROVED** [review]

File: libs/cms/src/lib/cms.contentful.service.ts (hunk 2 above)
Comment (Ballioli) [reply libs/cms/src/lib/cms.contentful.service.ts:219]: Já mætti alveg vera type Locale = 'is' | 'en'

File: apps/web/screens/UniversitySearch/LandingPage.tsx (hunk 1 above)
Comment (Ballioli) [reply apps/web/screens/UniversitySearch/LandingPage.tsx:66]: Has not been a problem for this case,  but I didn't know we had a sortAlpha in our utils, good to know!

**Pull request merged**